standalone/graph.js

```diff
@@ -128,70 +130,77 @@ function drawGraph(qlog, settings){
 			let packetAckedList = [];
 			let packetLostList = [];
 			
-			let packetsReceived = multistreamDictionary.get("TRANSPORT").get("PACKET_RECEIVED") || []; // || [] defaults to an empty array if there are no events of that type present in the log
-			for( let packet of packetsReceived ){
-				
-				let data = packet.details;
-				
-				if( !data.frames )
-					continue;
-				
-				let ackFrames = [];
-				for( let frame of data.frames ){
-					if( frame.frame_type == "ACK" )
-						ackFrames.push( frame );
-				}
-				
-				if( ackFrames.length == 0 )
-					continue;
-					
-				// now we have the ACK frames. These are composed of ACK blocks, each ACKing a range of packet numbers 
-				// we go over them all, look them up individually, and add them to packetAckedList
-				for( let frame of ackFrames ){
-					for( let range of frame.acked_ranges ){
-						let from = parseInt( range[0] );
-						let to = parseInt( range[1] ); // up to and including
-						
-						// ackedNr will be the ACKed packet number of one of our SENT packets here 
-						for( let ackedNr = from; ackedNr <= to; ++ackedNr ){
-							// find the originally sent packet 
-							let sentPacket = packetSentList[ ackedNr ]; 
-							if( !sentPacket ){
-								console.error("Packet was ACKed that we didn't send... ignoring", ackedNr, frame, packet);
-								continue;
-							}
-                            
+			let packetsReceived =[];
```

Comment:
similar remarks here: did you use the same indentation method as the rest of the file? Looks garbled, at least on github.

Reply:
I'm not sure why it looks so strange on Github, it looks fine to me on my IDE. 
![image](https://user-images.githubusercontent.com/17273720/61474477-99350a00-a93d-11e9-9472-3622c472a40d.png)

I followed the same indentation method.